src/tri/_tri.cpp

```diff
@@ -2055,17 +2056,4 @@ TrapezoidMapTriFinder::Trapezoid::set_upper_right(Trapezoid* upper_right_)
     upper_right = upper_right_;
     if (upper_right != 0)
         upper_right->upper_left = this;
-}
-
-
-
-RandomNumberGenerator::RandomNumberGenerator(unsigned long seed)
-    : _m(21870), _a(1291), _c(4621), _seed(seed % _m)
-{}
-
-unsigned long
-RandomNumberGenerator::operator()(unsigned long max_value)
-{
-    _seed = (_seed*_a + _c) % _m;
-    return (_seed*max_value) / _m;
-}
+}
```

Comment:
Please end the file with an empty line. That will fix the pre-commit issue. (The pyside6 issue is fixed in #24158.)